pkg/frontend/querymiddleware/shard_active_series_test.go

```diff
@@ -423,7 +425,7 @@ func Test_shardActiveSeriesMiddleware_RoundTrip_concurrent(t *testing.T) {
 }
 
 func Test_shardActiveSeriesMiddleware_mergeResponse_contextCancellation(t *testing.T) {
-	s := newShardActiveSeriesMiddleware(nil, mockLimits{}, log.NewNopLogger()).(*shardActiveSeriesMiddleware)
+	s := newShardActiveSeriesMiddleware(nil, true, mockLimits{}, log.NewNopLogger()).(*shardActiveSeriesMiddleware)
```

Comment:
As long as both code paths are still there I think it makes sense to run this test case for both, wdyt?

Reply:
sounds fair! 👍